src/components/QuestDetails/components/StreakProgress/index.tsx

```diff
@@ -152,14 +144,10 @@ export default function StreakProgress({
             {` / ${requiredStreakInDays} ${i18n.days}`}
           </div>
         </div>
-        <div className={styles.progressIconsContainer}>
-          {lightningBoltIcons.map((filled, index) => (
-            <LightningBolt
-              className={classNames(styles.streakIcon, getFilledClass(filled))}
-              key={`lightningBolt:${index}`}
-            />
-          ))}
-          <LightningBolt />
+        <div className={styles.syncContainer}>
+          <Button type="secondaryGradient" onClick={onSync}>
```

Comment:
the secondary gradient doesn't seem to match the designs. it's kind of small ui issue but I am wondering if we should update secondaryGradient to match the design or introduce a new gradient button type

Reply:
we can update